JSON.md

```diff
@@ -16,6 +16,91 @@ All these 3 share 1 standard format, commonly called `cfbs.json`.
 Used by `cfbs add` and `cfbs search`, when index key is present in `cfbs.json` in the current working directory.
 When adding a module by URL, which has a `cfbs.json` inside of it, the index in that file should be ignored.
 
+## The process of building modules from a project into a policy set
+
+When you build a project with the `cfbs build` command, it loops through all modules in the project (`"build"` key), in order.
+Within each module it runs the individual build steps, specified in the `"steps"` key.
+
+As an example, here is a local policy file module with 1 build step:
+
+```json
+{
+  "name": "./policy.cf",
+  "description": "Local policy file added using cfbs command line",
+  "tags": ["local"],
+  "added_by": "cfbs add",
+  "steps": [
+    "copy ./policy.cf services/cfbs/policy.cf",
+    "policy_files ./policy.cf",
+    "bundles my_bundle"
+  ]
+}
+```
+
+In the example above, the `policy.cf` file is copied to `out/masterfiles/services/autorun/policy.cf`.
```

Comment:
```suggestion
The 3 build steps above achieve 3 distinct things:
1. The policy file `policy.cf` is included in the policy set. It will be deployed to `/var/cfengine/masterfiles/services/cfbs/policy.cf` on the hub.
2. The path to `policy.cf` is added to `"inputs"` making cf-agent and other binaries aware of it and parse it.
3. The bundle `my_bundle` within `policy.cf` is added to the bundle sequence, making cf-agent find the correct bundle to run and, in the end, enforcing the promises within (commonly called the entry point of a policy file).

```